modules/aws_mysql/aws-mysql.yaml

```diff
@@ -39,6 +39,11 @@ inputs:
     validator: bool(required=False)
     description: Add deletion protection to stop accidental db deletions
     default: false
+  - name: identifier
+    user_facing: true
+    validator: str(required=False)
+    description: Identifier for the RDS instance.
```

Comment:
let's add something to explain the default value.
_If not set, use a generated unique name based on the layer and module name_